portfolio/src/main/webapp/servlet.js

```diff
@@ -0,0 +1,9 @@
+/** Fetches the current date from the server and adds it to the page. */ 
+
```

Comment:
Remove the blank line per https://google.github.io/styleguide/jsguide.html#jsdoc-general-form

Reply:
Removed! Thank you!